src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -41,6 +41,15 @@ export const OptionsListStrings = {
       i18n.translate('controls.optionsList.editor.hideExclude', {
         defaultMessage: 'Allow selections to be excluded',
       }),
+    getHideExistsQueryTitle: () =>
+      i18n.translate('controls.optionsList.editor.hideExistsQuery', {
+        defaultMessage: 'Allow exists query',
+      }),
+    getHideExistsQueryTooltip: () =>
+      i18n.translate('controls.optionsList.editor.hideExistsQueryTooltip', {
+        defaultMessage:
+          'The exists query will only return documents that contain an indexed value for the given field.',
+      }),
```

Comment:
Need input here -- FIX THIS

Reply:
How about:

`Returns the documents that contain an indexed value for the field.`

Reply:
That's amazing, thank you so much! 👍

Reply:
One thing that this text doesn't properly cover is that this setting **turns on and off** that ability.

Maybe `Allow the user to create an exists query which returns all documents that contain an indexed value for this field`?

Reply:
I had auto-merge on cause I thought everything was good, sorry ( ͡ಠ ʖ̯ ͡ಠ)

I do prefer this wording though, so willing to open a very quick PR to update this if @KOTungseth is good with this more robust wording 🎉

Reply:
No worries, and this is more up to @KOTungseth anyway!

Reply:
How about:

`When enabled, allows you to create an exists query, which returns all documents that contain an indexed value for the field.`

Reply:
@KOTungseth Could we possibly drop the `When enabled`? The tooltip text for the `"Ignore timeout for results"` option starts simply with `"Wait to display results until the list is complete. (...)"` and doesn't mention enabling, so just thinking about consistency between the options in this flyout 👍 

So, the text would be`"Allows you to create an exists query, which returns all documents that contain an indexed value for the field."`

Reply:
Our writing guidelines are to eliminate "When enabled" and instead simply describe the action when the toggle is enabled.